src/components/widgets/search-panel/SearchFilterPanel.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+import { Col, Row } from "reactstrap";
+import { FilterPanel } from "components/panels";
+import SelectFilter from "../../Filters/SelectFilter";
+import { getSelectRating } from "../../../utils";
+import { InputFilter } from "../../Filters";
+import { ICandidateFilters, OptionType } from "../../../types/types";
+import { addFilter } from "../../../redux/filters";
+
+interface Props {
+    title: string
+    statusSelectOptions: OptionType[]
+    callback: (filters: ICandidateFilters) => void;
+}
+
+export const SearchFilterPanel = ({ callback, title, statusSelectOptions }: Props) => {
```

Comment:
I would call it CandidateSearchFilterPanel and I would place it somewhere in components/panels.